app/routers/session.py

```diff
@@ -48,36 +49,30 @@ async def create_session(
 
     """
     new_session = CreateSessionResponse()
-
     await cache.set(new_session.session_id, session.model_dump_json())
     return new_session
 
 
-@ROUTER.get(
-    "/",
-    response_model=ListSessionsResponse,
-    responses={status.HTTP_404_NOT_FOUND: {"model": HTTPNotFoundError}},
-)
+@ROUTER.get("/", response_model=ListSessionsResponse)
 async def list_sessions(
     cache: TRedisPlugin,
 ) -> ListSessionsResponse:
     """Get all session keys"""
-    keylist = []
+    keylist: list[Union[str, bytes]] = []
```

Comment:
```suggestion
    keylist: list[str] = []
```
Since the key is decoded if `bytes`, this will just be a list of strings.